core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/WindowGroupLimitParser.scala

```diff
@@ -29,7 +29,7 @@ case class WindowGroupLimitParser(
   val fullExecName: String = node.name + "Exec"
   // row_number() is currently not supported by the plugin (v24.04)
```

Comment:
nit: We should update this comment as `row_number()` is now supported.

Reply:
Done